packages/block-editor/src/components/block-tools/use-block-toolbar-popover-props.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useState } from '@wordpress/element';
+
+const TOOLBAR_HEIGHT = 72;
+const DEFAULT_PROPS = { __unstableForcePosition: true, __unstableShift: true };
+const RESTRICTED_HEIGHT_PROPS = {
+	__unstableForcePosition: false,
+	__unstableShift: false,
+};
```

Comment:
Let's add some comments here explaining the expected behaviour when setting these options (e.g with "restricted" mode, the toolbar won't shift when there's no space, but it will flip instead)

Reply:
Agreed - I've improve the comments. 👍